rstudio/rstudio
# show custom icon for rproj files on Windows

### Intent

On Windows, project files (*.rproj) should have the same custom icon with Electron that they did with Qt. This makes them much easier to spot in Windows Explorer.

Addresses #12246

### Approach

Embed the icon in rstudio.exe (which is really just electron.exe) using ResourceHacker tool. I did a test build in my own Windows docker environment to confirm that this tool will execute in a headless environment.

### Automated Tests

None

### QA Notes

Try installing a build of RStudio Electron prior to this fix, and browse to a folder containing an .rproj file in Explorer, and note the blank icon.

After installing a build with this fix, verify that the icon for R project files (*.rproj) in Windows Explorer is as follows:

![icon](https://user-images.githubusercontent.com/10569626/199335028-7d120013-122b-4c26-bc71-90a9614c27a0.png)

Note that Explorer does some icon caching, so it may not always update immediately. Worst case, try logging out and back in.

### Checklist
